ivadomed/keywords.py

```diff
@@ -35,7 +35,6 @@ class LoaderParamsKW:
     CONTRAST_PARAMS = "contrast_params"
     MULTICHANNEL = "multichannel"  # boolean key that is used to change the configuration file ever slightly.
     EXTENSIONS = "extensions"
-    TARGET_GROUND_TRUTH = "target_ground_truth"
```

Comment:
Removed an obsolete/unused keyword that was added in a separate PR to indicate ground truth.

Reply:
Being split into clean up PR.